sig-scalability/annual-report-2022.md

```diff
@@ -4,7 +4,7 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
+   - We helped validating scalability and reliability impact of many features across whole year.
```

Comment:
Can we add some specific examples here? Like AP&F improvements, GOGC tuning with golang 1.19, etc

Reply:
There are also some improvements the SIG made (or wrote keps for) that may be worth calling out here? Not at all exhaustive, but few examples: 
- streaming list for watch - https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/3157-watch-list/kep.yaml
- page size progressing for lists with selector - https://github.com/kubernetes/kubernetes/pull/108569
- optimizing apiserver gzip - https://github.com/kubernetes/kubernetes/issues/112296
- multiple bug fixes/improvements around watch/apf performance

These may be overlapping with sig-apimachinery, but doesn't mean it's not scalability work.